loader/windows/icd_windows.c

```diff
@@ -279,7 +282,19 @@ void *khrIcdOsLibraryGetFunctionAddress(void *library, const char *functionName)
     {
         return NULL;
     }
+#if defined(_MSC_VER) && !defined(__clang__)
+#pragma warning( push )
+#pragma warning( disable : 4152 )
+#elif defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpedantic"
+#endif
     return GetProcAddress( (HMODULE)library, functionName);
+#if defined(_MSC_VER) && !defined(__clang__)
+#pragma warning( pop )
+#elif defined(__clang__)
+#pragma clang diagnostic pop
+#endif
```

Comment:
I would try using:
```suggestion
    return (void *)(intptr_t)GetProcAddress( (HMODULE)library, functionName);
```
here. I would be consistent with what we do at other place in the code. You may require an additional include.